spacepy/datamodel.py

```diff
@@ -1834,8 +1834,44 @@ def dmcopy(dobj):
     else:
         return copy.copy(dobj)
 
-def createISTPattrs(datatype, ndims=1, vartype=None, units=None, NRV=False):
+def createISTPattrs(datatype, ndims=1, vartype=None, units=' ', NRV=False):
     '''Return set of unpopulated attributes for ISTP compliant variable
+
+    Parameters
+    ----------
+    datatype : str
+        datatype of variable to reate metadata for (data|support_data|metadata)
```

Comment:
```suggestion
    datatype : {'data', 'support_data', 'metadata'}
        datatype of variable to create metadata for.
```

See "parameters" in the [numpydoc docs](https://numpydoc.readthedocs.io/en/latest/format.html#sections)